language/pl-PL/com_users.ini

```
@@ -4,119 +4,159 @@
-COM_USERS_EMAIL_REGISTERED_WITH_ADMIN_ACTIVATION_BODY_NOPW="Sz. P. {NAME},\n\nWitaj, Dziękujemy za rejestrację na {SITENAME}. Twoje konto zostało już utworzone, ale zanim będzie można z niego korzystać, wymaga uaktywnienia.\n\nW tym celu naciśnij poniższy odnośnik albo skopiuj go i wklej w pasku adresu Twojej przeglądarki:\n {ACTIVATE}\n\nNa tej podstawie administrator witryny zostanie poproszony o zatwierdzenie Twojego konta.\n\nO zatwierdzeniu konta otrzymasz powiadomienie.\n\nGdy Twoje konto zostanie zatwierdzone, będzie możliwe logowanie się na stronie {SITEURL}, przy pomocy poniższej nazwy użytkownika:\n\nUsername: {USERNAME}"
-COM_USERS_EMAIL_USERNAME_REMINDER_BODY="Witaj,\n\nPrzypominamy na Twoją prośbę nazwę konta użytkownika na {SITENAME}.\n\nTwoja nazwa brzmi\n\nUżytkownik: {USERNAME}.\n\nJeśli chcesz się teraz zalogować na swoje konto, wybierz odnośnik poniżej.\n\n{LINK_TEXT} \n\nPozdrawiamy!"
-COM_USERS_EMAIL_USERNAME_REMINDER_SUBJECT="Nazwa Twojego konta użytkownika na {SITENAME}"
-COM_USERS_ERROR_SECRET_CODE_WITHOUT_TFA="Wprowadzono tajny kod, ale weryfikacja dwuskładnikowa nie jest włączona w koncie użytkownika. Jeśli chcesz korzystać z kodu weryfikacyjnego, aby wzmocnić zabezpieczenie logowania, edytuj swój profil użytkownika i uaktywnij weryfikację dwuskładnikową."
+COM_USERS_EMAIL_PASSWORD_RESET_BODY="Witamy,\n\nOtrzymaliśmy prośbę o zresetowanie hasła dla Twojego konta na {SITENAME}. Aby zresetować hasło, musisz przesłać ten kod weryfikacyjny, aby sprawdzić, czy żądanie było uzasadnione.\n\nTwój kod weryfikacyjny to {TOKEN}\n\nWybierz poniższy adres URL i kontynuuj resetowanie hasła.\n\n{LINK_TEXT} \n\nDziękujemy."
+COM_USERS_EMAIL_PASSWORD_RESET_SUBJECT="Twoja prośba o zresetowanie hasła na {SITENAME}"
+COM_USERS_EMAIL_REGISTERED_BODY="Witaj {NAME},\n\nDziękujemy za zarejestrowanie się na {SITENAME}.\n\nMożesz zalogować się teraz do {SITEURL}, używając następującej nazwy użytkownika i hasła:\n\nNazwa użytkownika: {USERNAME}\nHasło: {PASSWORD_CLEAR}"
+COM_USERS_EMAIL_REGISTERED_BODY_NOPW="Witaj {NAME},\n\nDziękujemy za zarejestrowanie się na {SITENAME}.\n\nMożesz zalogować się teraz do {SITEURL}, używając nazwy użytkownika i hasła podanych podczas rejestracji."
+COM_USERS_EMAIL_REGISTERED_NOTIFICATION_TO_ADMIN_BODY="Witaj administratorze, \n\nNowy użytkownik '{NAME}', nazwa użytkownika: '{USERNAME}', zarejestrował się na {SITEURL}."
```

To zwrot do adresata
```suggestion
COM_USERS_EMAIL_REGISTERED_NOTIFICATION_TO_ADMIN_BODY="Witaj Administratorze, \n\nNowy użytkownik '{NAME}', nazwa użytkownika: '{USERNAME}', zarejestrował się na {SITEURL}."
```